src/components/cc-addon-backups/cc-addon-backups.js

```diff
@@ -326,7 +309,6 @@ export class CcAddonBackups extends LitElement {
    */
   _renderBackups(state) {
     const skeleton = state.type === 'loading';
-    const areBtnsDisabled = state.type === 'loading' || this._overlayType != null;
```

Comment:
we don't need to disable buttons when dialog is open (`overlayType` is `null`) because the `cc-dialog` component traps the focus anyway.
we don't need to disable buttons in skeleton since they don't do anything while in skeleton anything.